lib/payment_processor/braintree/one_click.rb

```diff
@@ -72,7 +72,11 @@ def params_for_action
     def make_sale
       @make_sale ||= ::Braintree::Transaction.sale(
         payment_method_token: payment_options.token,
-        amount: payment_options.amount
+        amount: payment_options.amount,
+        merchant_account_id: payment_options.currency,
```

Comment:
yeah doing that now (wanted to get get the express donations merged as that also has some extra specs that i didn't want conflicts with). btw, this is wip! (adding tag now)